src/value.rs

```diff
@@ -222,7 +323,7 @@ macro_rules! type_to_value {
 }
 
 macro_rules! type_to_box_value {
-    ( $type: ty, $name: ident, $col_type: expr ) => {
+    ( $type: ty, $name: ident, $col_type: expr, $array_type: expr ) => {
```

Comment:
Nitpick: a trivial refactor
- https://github.com/ikrivosheev/sea-query/pull/1

Reply:
Thank you!